app/models/patient.rb

```diff
@@ -369,6 +370,28 @@ class Patient < ApplicationRecord
           where.not(not_eligible_criteria.first.arel.exists)
         end
 
+  # Patients that are in the given academic year, across all the teams and who
+  # could potentially be given a vaccination, irrespenctive of eligibility for
+  # any particular vaccine.
+  scope :enrolled_in_academic_year,
+        ->(academic_year = AcademicYear.pending) do
```

Comment:
I'm wondering if it's better to be explicit and avoid a default academic year:

1. The value will be evaluated once at runtime, if the application continues to run over a rollover we might see odd behaviour.
2. When I was reviewing other parts of the code before getting to this I kept thinking, "which academic year is this checking?".